docs/core/install.md

```diff
@@ -1,66 +1,71 @@
 # Installation
 
-## conda or better yet [mamba](https://mamba.readthedocs.io/en/latest/user_guide/mamba.html) - easy
+### conda or better yet [mamba](https://mamba.readthedocs.io/en/latest/user_guide/mamba.html) - easy
 
-We do not have official conda recipes (yet!), so to install with conda or mamba you will need to clone the
-[fairchem](https://github.com/FAIR-Chem/fairchem) and run the following from inside the repo directory to create an environment with all the
-necessary dependencies.
+We do not have official conda recipes (yet!); in the meantime you can use the
+following environment yaml files for CPU [env.cpu.yml](https://raw.githubusercontent.com/FAIR-Chem/fairchem/main/packages/env.cpu.yml)
+and GPU [env.gpu.yml](https://raw.githubusercontent.com/FAIR-Chem/fairchem/main/packages/env.gpu.yml) to easily set up a
+working environment and install `fairchem-core`.
 
-1. Create a *fairchem* environment
+1. Create an environment to install *fairchem*
    1. **GPU**
 
       The default environment uses cuda 11.8, if you need a different version you will have to edit *pytorch-cuda* version
       accordingly.
       ```bash
-      conda env create -f packages/env.gpu.yml
+      conda env create -f env.gpu.yml
       ```
 
    2. **CPU**
       ```bash
-      conda env create -f packages/env.cpu.yml
+      conda env create -f env.cpu.yml
```

Comment:
should this be packages/env.cpu.yml , i dont see env.cpu.yml in the root

Reply:
I wrote it like this now so that it does not imply you have to clone the repo. I linked the env.yml files right above

Reply:
Awesome! I missed that! LGTM!